src/Chainweb/Chainweb.hs

```diff
@@ -342,7 +342,7 @@ mempoolConfig enableReIntro = Mempool.InMemConfig
     maxRecentLog
     enableReIntro
   where
-    blockGasLimit = 1000000               -- TODO: policy decision
+    blockGasLimit = 100000               -- TODO: policy decision
```

Comment:
What are each of these numbers doing? This one has decreased, but the below has increased by an order of magnitude. Could you explain?

Reply:
Ah n/m you'e explained it.